gnovm/stdlibs/runtime/runtime.gno

```diff
@@ -0,0 +1,35 @@
+// Package runtime allows access to the GnoVM's runtime information.
+package runtime
+
+func NewReadOnlyBanker() chain.Banker {
+}
+
+func NewLimitedSendBanker() chain.Banker {
+}
+
+func NewSendBanker() chain.Banker {
+}
+
+func NewIssueBanker() chain.Banker {
+}
+
+func AssertOriginCall() {
+}
+
+func CallerRealm() chain.Realm {
+}
+
+func CurrentRealm() chain.Realm {
+}
+
+func Deposits() chain.Coins {
```

Comment:
We need to make a distinction between prevrealm deposits and origin deposits; rather, you only need to read the immediate previous realm deposit; I'm not sure of a usecase where origin deposit could be needed.